pkg/controllers/jobflow/constant.go

```diff
@@ -0,0 +1,10 @@
+package jobflow
```

Comment:
please add the copyright in your each file :)

Reply:
thanks.